main.go

```diff
@@ -177,11 +177,10 @@ var initCmd = &cli.Command{
 
 		// create the package directory with a temporary file so that go recognises it as a package
 		// and autobinding doesn't error out
-		tmpPackageNameFile := "graph/model/_tmp_gqlgen_init.go"
+		tmpPackageNameFile := "graph/model/doc.go"
 		if err := initFile(tmpPackageNameFile, "package model"); err != nil {
 			return err
 		}
-		defer os.Remove(tmpPackageNameFile)
```

Comment:
removed generated file by this line

Reply:
Why does the temp file need to persist? I assume the leading underscore is the reason go isn't recognising the package?

Reply:
I don't know what you're talking about.
The reason is that the "package model" file that should have been generated was treated as a temporary file. To solve the problem you wanted to solve, you should not delete. Please try it on your own

Reply:
I mean we _can_ keep the doc.go around. I just don't understand why it's necessary. I can't reproduce the issue in #1849, and leaving this line in place works fine for me running both `init` and `generate`

Reply:
@mtibben you can reproduce [this way](https://github.com/99designs/gqlgen/issues/1849#issuecomment-1021621837).

Reply:
I don't understand what this meant.

> I mean we can keep the doc.go around

Do you mean removed `os.Remove` line?

> I just don't understand why it's necessary.

You don't know what you want to solve.

Reply:
this fix only init sub command

Reply:
Yes I can't reproduce the original error. The reason I re-opened #1849 was because the instructions in the quick start to manually add the `doc.go` file seemed very strange to me and I wanted the root cause addressed.

This works fine for me
```
$ mkdir example
$ cd example
$ go mod init example
go: creating new go.mod: module example
$ printf '// +build tools\npackage tools\nimport _ "github.com/99designs/gqlgen"' | gofmt > tools.go

$ go get -d github.com/99designs/gqlgen@master
go get: added github.com/99designs/gqlgen v0.16.1-0.20220301034206-5236fb096802
go get: added github.com/agnivade/levenshtein v1.1.0
go get: added github.com/cpuguy83/go-md2man/v2 v2.0.1
go get: added github.com/gorilla/websocket v1.4.2
go get: added github.com/hashicorp/golang-lru v0.5.0
go get: added github.com/matryer/moq v0.2.3
go get: added github.com/mitchellh/mapstructure v1.2.3
go get: added github.com/russross/blackfriday/v2 v2.1.0
go get: added github.com/urfave/cli/v2 v2.3.0
go get: added github.com/vektah/gqlparser/v2 v2.4.0
go get: added golang.org/x/mod v0.5.1
go get: added golang.org/x/sys v0.0.0-20211019181941-9d821ace8654
go get: added golang.org/x/tools v0.1.9
go get: added golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
go get: added gopkg.in/yaml.v2 v2.2.8
$ go mod tidy
$ go run github.com/99designs/gqlgen init

Creating gqlgen.yml
Creating graph/schema.graphqls
Creating server.go
Generating...

Exec "go run ./server.go" to start GraphQL server
$ go run github.com/99designs/gqlgen generate
$ echo $?
0
```

Reply:
@mtibben because you used the latest version of gqlgen (master branch). reproduce with v0.16.0